paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -353,10 +355,26 @@ def autoscale_marathon_instance(marathon_service_config, marathon_tasks, mesos_t
         **autoscaling_params
     )
 
-    new_instance_count = marathon_service_config.limit_instance_count(current_instances + autoscaling_amount)
+    # Limit downscaling by 30% of current_instances until we find out what is
+    # going on in such situations
+    new_instance_count = max(current_instances + autoscaling_amount, int(current_instances * 0.7))
+
+    new_instance_count = marathon_service_config.limit_instance_count(new_instance_count)
     if new_instance_count != current_instances:
         write_to_log(
             config=marathon_service_config,
+            line=('decision_policy %s, metrics_provider %s, utilization %.2f,'
+                  ' error %.2f, autoscaling_amount %d, limited_downscaling %r'
```

Comment:
Even with this debug output, I don't think it will have the actual input data we need. We don't have the inputs to the utilization. (all the other stuff can more or less be calcualted after the fact